hypershift-operator/controllers/hostedcluster/metrics/metrics.go

```diff
@@ -60,20 +60,24 @@ var (
 		Help: "Indicates the operator will skip the aws resources deletion",
 		Name: SkippedCloudResourcesDeletionName,
 	}, []string{"namespace", "name"})
+
+	metricsVecs = []*prometheus.MetricVec{
+		HostedClusterDeletionDuration.MetricVec,
+		HostedClusterGuestCloudResourcesDeletionDuration.MetricVec,
+		HostedClusterAvailableDuration.MetricVec,
+		HostedClusterInitialRolloutDuration.MetricVec,
+		ClusterUpgradeDuration.MetricVec,
+		LimitedSupportEnabled.MetricVec,
+		SilenceAlerts.MetricVec,
+		ProxyConfig.MetricVec,
+		SkippedCloudResourcesDeletion.MetricVec,
+	}
 )
 
 func init() {
-	metrics.Registry.MustRegister(
```

Comment:
what's the rational for this change?

Reply:
Factorisation. The `StopReporting` method is reusing the list.